### PR TITLE
Twiml url

### PR DESCRIPTION
**Pull request by norkans7**
Adding the action in the url will allow us to support more action if needed in the future, like if the delivery status are improved or the supporting voice we can add easily status or answer action without breaking what already exists and configured for the existing channels.

### CODE REVIEW

File: templates/channels/channel_configuration.haml [hunk 1]
```diff
@@ -287,7 +287,7 @@
         Incoming messages for this channel will be sent to this endpoint.
 
     %code
-      https://{{ domain }}{% url 'handlers.twiml_api_handler' object.uuid %}
+      https://{{ domain }}{% url 'handlers.twiml_api_handler' 'received' object.uuid %}
```
Comment (nicpottier) [inline templates/channels/channel_configuration.haml]: So I think that actually the TwiML API stuff is used more for voice than SMS.. I'm guessing those endpoints send in the actions the same way Twilio does.

**Comment by norkans7** [comment]
@teehamaral please check this before it is merged, I think it is better to update the URL to include  'received' before there are many TWIML type channels. Let us know what you think

**Comment by teehamaral** [comment]
I also think that TwiML API would be used more for voice than SMS, but can be that adding 'received' to URL, would be more generic to both channels. In this case, the way that action is being obtained in handler must be changed too.

**Comment by norkans7** [comment]
I think the URLs are configured by the user on TWIML REST API instance/Twilio set those with explicit actions seems better to me 

so we will be handling the calls with the action answer(currently no action is used)

I think those URL being different help the user understand he needs to configure them both instead of one twice

**Comment by teehamaral** [comment]
Right, I understood. But you can review this line: https://github.com/nyaruka/rapidpro/blob/9ae95a3e2ade9dd091875859f4cb90c74defa619/temba/channels/handlers.py#L117, because "action" also can come in the kwargs instead of GET.

**Comment by norkans7** [comment]
I think there should be no default action as it is there so the action is from the kwargs as the url specifies and that line should move up to https://github.com/nyaruka/rapidpro/blob/9ae95a3e2ade9dd091875859f4cb90c74defa619/temba/channels/handlers.py#L46 before handling the call so we only handle the call for the proper action like answer

The current behavior works perfectly however I added to make is consistent with how other channels URLs have always action in the URL

**Pull request closed without merging**